cypress/integration/transfer/reorder-with-buttons.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Reorder items in the selected list
+
+  Background:
+    Given reordering of items is enabled
+
+# I created two scenarios for up and down reordering because I wanted to incorporate a check for 'is first' or 'is last' and it seemed overly ambiguous to write this in a single scenario. Or, is it enough that I have included the 'Disable move buttons' scenarios at the bottom of this file, so that doesn't need to be handled in the basic scenarios at the top?
+
+  Scenario: Reorder items upwards using buttons
+    Given a selected list with one or more items
+    And a highlighted item in the selected list
+    And the highlighted item is not the first item in the list
+    When the 'move up' button is clicked
+    Then the highlighted item moves up one place in the selected list
+
+  Scenario: Reorder items downwards using buttons
+    Given a selected list with one or more items
+    And a highlighted item in the selected list
+    And the highlighted item is not the last item in the list
+    When the 'move down' button is clicked
+    Then the highlighted item moves down one place in the selected list
+
+  Scenario: Disable reorder buttons when no items highlighted
+    Given a selected list with one or more items
+    When no items are highlighted in the list
+    Then the 'move up' and 'move down' reorder buttons are disabled
```

Comment:
`When` should be `Given`/`And`

The buttons themselves are not disabled. The user can still press them, so I think it's better to change the `Then` to

```feature
When the `move up` is pressed
Then the order remains the same
```

And then use `Scenario Outline` to make sure both buttons are being tested

Reply:
This is technically correct, as per your comment here https://github.com/dhis2/ui-core/pull/607#discussion_r349965919, so I'll revert this back to the original scenario.

Reply:
Reverted to original scenario to reflect that the buttons are actually disabled.